cmake/KokkosCore_config.h.in

```diff
@@ -52,6 +52,7 @@
 #cmakedefine KOKKOS_OPT_RANGE_AGGRESSIVE_VECTORIZATION  // deprecated
 #cmakedefine KOKKOS_ENABLE_AGGRESSIVE_VECTORIZATION
 #cmakedefine KOKKOS_ENABLE_IMPL_MDSPAN
+#cmakedefine KOKKOS_ENABLE_IMPL_REF_COUNT_BRANCH_UNLIKELY
```

Comment:
I don;t care so much if that is something the user can specify with generated makefiles but you do need to add an entry from the makefile.kokkos as well to match this.

Reply:
We probably want to include this in the core print_configuration info

Reply:
See here https://github.com/kokkos/kokkos/blob/af933fe32d94fbc180adae8855979bfaec0aec01/Makefile.kokkos#L1232

Reply:
Made the change